components/builder-worker/src/runner/mod.rs

```diff
@@ -314,10 +304,14 @@ impl Runner {
         self.log_pipe().pipe(&mut child)?;
         let exit_status = child.wait().expect("failed to wait on child");
         debug!("build complete, status={:?}", exit_status);
+        fs::rename(self.workspace.src().join("results"), self.workspace.out())?;
```

Comment:
Is there a case where the results dir is not present when a build fails, and therefore the rename fails? In that case, we won't send an exit status code in the Err.  If we want to keep doing that, might be better to map the rename err to an exit status code.